docs/content/users/quickstart.md

```diff
@@ -323,7 +323,7 @@ The Laravel project type can be used for [Lumen](https://lumen.laravel.com/) lik
     ```bash
     mkdir my-laravel-app
     cd my-laravel-app
-    ddev config --project-type=laravel --docroot=public
+    ddev config --project-type=laravel --docroot=public --php-version=8.2
```

Comment:
Just a note that if using an earlier version of DDEV `mysql:8.0` seems to be required:

```
    ddev config --project-type=laravel --docroot=public --php-version=8.2 --database=mysql:8.0
```

Reply:
I think that using `--php-version=8.2` is fine, but it makes me wonder whether we should show explicit defaults then 1) it's already the DDEV default and 2) we're already overriding it in the Laravel handling in DDEV.

Reply:
I believe this is good when people use the latest docs without even noticing it.